wurst/systems/entities/bosses/AncientHydra.wurst

```diff
@@ -59,6 +59,9 @@ public class AncientHydra extends Hostile
             end)
 
         if frostReady
```

Comment:
Does this only happen if `frostReady` is true and there isn't a nearby unit to use breath of frost (or the equivalent ability) on?

Reply:
Yes